Added the new method for disabling the bundled CLI version in 2020 an… (#715)
OctopusDeploy/docs: docs/deployment-examples/azure-deployments/running-azure-powershell/configuring-the-version-of-the-azure-powershell-cli.md

If you wish to use a different version, you can install the Azure PowerShell CLI on your Octopus Server, and configure Octopus to use the installed version instead. 

The procedure to configure this differs depending on which version of Octopus Deploy you are using:

## Octopus 2020.1 or newer {#ConfiguringtheversionoftheAzurePowerShellmodules-Octopus2020.1(ornewer)}

> Within the PowerShell Azure step, there will be an option called "Azure Tools". Toggle the setting to **Use Azure Tools pre-installed on the worker**.

Reviewer: ```suggestion
Within the PowerShell Azure step, is an option called "Azure Tools". Toggle the setting to **Use Azure Tools pre-installed on the worker**.
```
